### ISSUE TITLE
Invoking the TogglePattern through UIAutomation on ToggleSplitButton does not trigger the Command bound to the ToggleSplitButton

### ISSUE THREAD

**Issue by harvinders** [microsoft/microsoft-ui-xaml#6615]
### Describe the bug

Trying to toggle the `ToggleSplitButton` through the TogglePattern of UIAutomation successfully toggles the button, however any command that is bound to the control is not triggered.

### Steps to reproduce the bug

1. Create a ToggleSplitButton with a command with x:Bind
2. Open the application 
3. Open Accessibility Insights for Windows
4. Find the control and invoke the toggle command as shown below
![image](https://user-images.githubusercontent.com/8557046/149933407-7e06c1aa-cc89-48cc-9af4-332cb10222b4.png)
 5. button is toggled but command is not invoked

### Expected behavior

Command should be invoked, as there is no other way to invoke the command

### NuGet package version

WinUI 3 - Windows App SDK 1.0 (If you're seeing your issue in older previews of WinUI 3, please try this release)

### Windows app type

- [ ] UWP
- [X] Win32

### Device form factor

Desktop

### Windows version

October 2020 Update (19042)

**Comment by StephenLPeters** [comment]
@harvinders chigy FYI
